packages/web-twig/src/Resources/components/Dropdown/README.md

```diff
@@ -0,0 +1,41 @@
+# Dropdown
+
+This is Twig implementation of the [Dropdown] component.
+
+Basic example usage:
+
+```html
+<Dropdown>Dropdown</Dropdown>
+```
+
+Advanced example usage:
```

Comment:
Like in the [`Collapse` docs](https://spirit.supernova-docs.io/spirit/latest/components/collapse/twig.html), I'd like to see a real-world usage example with a dropdown trigger and JS binding. I think the `elementType` option can be just mentioned in the API table down there.

Reply:
Ok, I fixed that and added meaningful examples using Twig components. Eg. in Collapse Twig Readme should be Twig components as well. I'll fix that. Also added some more examples in tests.

Reply:
Oh, one more thing. We must use `twig` instead of `html` to specify the type of code example. If it is used `html` prettier will automatically lower `Button` to `button`. And I think this will be a problem in other places.

Reply:
Here is the fix I've mentioned - https://github.com/lmc-eu/spirit-design-system/pull/551